```
paul@Mjolnir:OpenRA+pr/11285$ make dependencies
Fetching StyleCopPlus from NuGet
Attempting to resolve dependency 'StyleCop.MSBuild (≥ 4.7.49.0)'.
Installing 'StyleCop.MSBuild 4.7.49.0'.
Successfully installed 'StyleCop.MSBuild 4.7.49.0'.
Installing 'StyleCopPlus.MSBuild 4.7.49.5'.
Successfully installed 'StyleCopPlus.MSBuild 4.7.49.5'.
Fetching StyleCop files from NuGet
Package "StyleCop.MSBuild" is already installed.
Fetching ICSharpCode.SharpZipLib from NuGet
Installing 'SharpZipLib 0.86.0'.
Successfully installed 'SharpZipLib 0.86.0'.
Fetching MaxMind.GeoIP2 from NuGet
Installing 'Newtonsoft.Json 7.0.1'.
Successfully installed 'Newtonsoft.Json 7.0.1'.
Installing 'MaxMind.Db 2.0.0'.
Successfully installed 'MaxMind.Db 2.0.0'.
Attempting to resolve dependency 'MaxMind.Db (≥ 2.0.0)'.
Attempting to resolve dependency 'Newtonsoft.Json (≥ 8.0.3)'.
Unable to resolve dependency 'Newtonsoft.Json (≥ 8.0.3)'.
make: *** [cli-dependencies] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -413,7 +413,6 @@ install-core: default
 	@$(INSTALL_PROGRAM) MaxMind.Db.dll "$(DATA_INSTALL_DIR)"
 	@$(INSTALL_PROGRAM) MaxMind.GeoIP2.dll "$(DATA_INSTALL_DIR)"
 	@$(INSTALL_PROGRAM) Newtonsoft.Json.dll "$(DATA_INSTALL_DIR)"
-	@$(INSTALL_PROGRAM) RestSharp.dll "$(DATA_INSTALL_DIR)"
 	@$(INSTALL_PROGRAM) SmarIrc4net.dll "$(DATA_INSTALL_DIR)"
 
 ifneq ($(UNAME_S),Darwin)

--- a/packaging/windows/OpenRA.nsi
+++ b/packaging/windows/OpenRA.nsi
@@ -104,7 +104,6 @@ Section "Game" GAME
 	File "${SRCDIR}\MaxMind.Db.dll"
 	File "${SRCDIR}\MaxMind.GeoIP2.dll"
 	File "${SRCDIR}\Newtonsoft.Json.dll"
-	File "${SRCDIR}\RestSharp.dll"
 	File "${SRCDIR}\GeoLite2-Country.mmdb.gz"
 	File "${SRCDIR}\eluant.dll"
 	File "${SRCDIR}\SmarIrc4net.dll"
@@ -207,7 +206,6 @@ Function ${UN}Clean
 	Delete $INSTDIR\MaxMind.Db.dll
 	Delete $INSTDIR\MaxMind.GeoIP2.dll
 	Delete $INSTDIR\Newtonsoft.Json.dll
-	Delete $INSTDIR\RestSharp.dll
 	Delete $INSTDIR\GeoLite2-Country.mmdb.gz
 	Delete $INSTDIR\KopiLua.dll
 	Delete $INSTDIR\soft_oal.dll

--- a/thirdparty/fetch-thirdparty-deps.ps1
+++ b/thirdparty/fetch-thirdparty-deps.ps1
@@ -38,15 +38,13 @@ if (!(Test-Path "ICSharpCode.SharpZipLib.dll"))
 if (!(Test-Path "MaxMind.GeoIP2.dll"))
 {
 	echo "Fetching MaxMind.GeoIP2 from NuGet."
-	./nuget.exe install MaxMind.GeoIP2 -Version 2.3.1 -ExcludeVersion
-	cp MaxMind.Db/lib/net40/MaxMind.Db.* .
+	./nuget.exe install MaxMind.GeoIP2 -Version 2.6.0 -ExcludeVersion
+	cp MaxMind.Db/lib/net45/MaxMind.Db.* .
 	rmdir MaxMind.Db -Recurse
-	cp MaxMind.GeoIP2/lib/net40/MaxMind.GeoIP2* .
+	cp MaxMind.GeoIP2/lib/net45/MaxMind.GeoIP2* .
 	rmdir MaxMind.GeoIP2 -Recurse
-	cp Newtonsoft.Json/lib/net40/Newtonsoft.Json* .
+	cp Newtonsoft.Json/lib/net45/Newtonsoft.Json* .
 	rmdir Newtonsoft.Json -Recurse
-	cp RestSharp/lib/net4-client/RestSharp* .
-	rmdir RestSharp -Recurse
 }
 
 if (!(Test-Path "SharpFont.dll"))

--- a/thirdparty/fetch-thirdparty-deps.sh
+++ b/thirdparty/fetch-thirdparty-deps.sh
@@ -49,18 +49,15 @@ fi
 
 if [ ! -f MaxMind.GeoIP2.dll ]; then
 	echo "Fetching MaxMind.GeoIP2 from NuGet"
-	get Newtonsoft.Json 7.0.1
-	get MaxMind.Db 1.1.0.0
-	get RestSharp 105.2.3
-	get MaxMind.GeoIP2 2.3.1
-	cp ./MaxMind.Db/lib/net40/MaxMind.Db.* .
+	get Newtonsoft.Json 8.0.3
+	get MaxMind.Db 2.0.0
+	get MaxMind.GeoIP2 2.6.0
+	cp ./MaxMind.Db/lib/net45/MaxMind.Db.* .
 	rm -rf MaxMind.Db
-	cp ./MaxMind.GeoIP2/lib/net40/MaxMind.GeoIP2* .
+	cp ./MaxMind.GeoIP2/lib/net45/MaxMind.GeoIP2* .
 	rm -rf MaxMind.GeoIP2
-	cp ./Newtonsoft.Json/lib/net40/Newtonsoft.Json* .
+	cp ./Newtonsoft.Json/lib/net45/Newtonsoft.Json* .
 	rm -rf Newtonsoft.Json
-	cp ./RestSharp/lib/net4-client/RestSharp* .
-	rm -rf RestSharp
 fi
 
 if [ ! -f SharpFont.dll ]; then

